go.mod

```diff
@@ -1,80 +1,194 @@
 module github.com/pulumi/crd2pulumi
 
-go 1.17
+go 1.18
 
 require (
-	github.com/pkg/errors v0.9.1
-	github.com/pulumi/pulumi/pkg/v3 v3.24.1
-	github.com/pulumi/pulumi/sdk/v3 v3.24.1
-	github.com/spf13/cobra v1.0.0
-	github.com/spf13/pflag v1.0.5
-	github.com/stretchr/testify v1.6.1
-	k8s.io/apimachinery v0.18.0
+	github.com/pkg/errors v0.9.1 // indirect
+	github.com/pulumi/pulumi/pkg/v3 v3.34.1
+	github.com/pulumi/pulumi/sdk/v3 v3.34.1
+	github.com/spf13/cobra v1.4.0
+	github.com/spf13/pflag v1.0.5 // indirect
+	github.com/stretchr/testify v1.7.2
+	k8s.io/apimachinery v0.24.1
 )
 
 require (
-	github.com/agext/levenshtein v1.2.1 // indirect
-	github.com/apparentlymart/go-textseg v1.0.0 // indirect
+	github.com/pulumi/pulumi/sdk v1.14.1
```

Comment:
Any idea why this is getting pulled in? Maybe a bad import path somewhere?

Reply:
Yep, it was the `v3` import you mentioned in your other comment